lang_de_DE.properties

```diff
@@ -156,9 +156,9 @@ version.manager.alert.time.minute.0.plural=%n Minute;%n Minuten
 version.manager.alert.time.hour=vor %0 Stunde
 version.manager.alert.time.hour.0.plural=%n Stunde;%n Stunden
 version.manager.alert.time.day=vor %0 Tag
-version.manager.alert.time.day.0.plural=%n Tag;%n Tagen
+version.manager.alert.time.day.0.plural=%n Tag;%n Tagen;%n
```

Comment:
German only has [two cardinal forms](https://unicode-org.github.io/cldr-staging/charts/37/supplemental/language_plural_rules.html#de): `one` and `other`, there's no need to add a third `%n`. Russian has [three or four forms](https://unicode-org.github.io/cldr-staging/charts/37/supplemental/language_plural_rules.html#ru) (we only use first three, that's why in `ru_RU` there are three segments in `.plural` lines).

Reply:
Hi @turikhay , you are absolutely right, I will correct this asap.